recipes/gnu-arm-embedded-toolchain/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+import os
+
+
+required_conan_version = ">=1.50.0"
+
+
+class GnuArmEmbeddedToolchain(ConanFile):
+    name = "gnu-arm-embedded-toolchain"
```

Comment:
I believe the name was changed to the Arm GNU Toolchain.

```suggestion
    name = "arm-gnu-toolchain"
```